### PR TITLE
Added Food Card Component

### PR DESCRIPTION
**Pull request by Naman-B-Parlecha**
Mohit anna 
i have made card component which we can reuse for list of items (when the data is ready) 
check if the design is proper or anything else to be added 

 --------------------------------------------------if this is proper--------------------------------------------------
-> next will work on:
i will see and make a data list of all the food items which we can use for the home page

### CODE REVIEW

File: src/App.jsx
Comment (mohit-nagaraj) [inline src/App.jsx:1]: Please make router n add routes, implement this in home page

File: src/components/Item.scss
Comment (mohit-nagaraj) [inline src/components/Item.scss:1]: great scss btw👍

**Review by mohit-nagaraj — CHANGES_REQUESTED** [review]
image
Appreciated if u cud further work on these, for further improvment:
1. Add object-fit: cover; for tht img
2. Make the description centered know, it doesnt look nice when title is centered and desc is not
3. Possible to make the counter variable centered?
4. Add price to show to users the price of tht 1 meal, or maybe u r doing it somewhere else? then ok

File: src/components/Item.scss
Comment (Naman-B-Parlecha) [reply src/components/Item.scss:1]: thanks boss learnt from u only

**Pull request closed without merging**